### PR TITLE
Feature/mobile menu background

### PR DESCRIPTION
**Pull request by cwefso**
This should work to make the mobile menu have the greyed-out background that we wanted.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,6 +6,8 @@
   </head>
   <meta name=‘viewport’ content=‘width=device-width, initial-scale=1’>
   <body class="purple-1">
+    <section class="background">
```
Comment (cwefso) [inline index.html:9]: I made a new section for the background here.

File: src/main.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ var saveIdeaButton = document.querySelector('.form-button');
 var userForm = document.querySelector('.form');
 var userNewBody = document.querySelector('.input-body');
 var userNewTitle = document.querySelector('.input-title');
+var background = document.querySelector('.background');
```
Comment (cwefso) [inline src/main.js:10]: query selector for that new section

File: src/main.js [hunk 3]
```diff
@@ -58,13 +59,15 @@ function showMobileMenu() {
   mobileMenuBody.classList.add('menu-body-mobile');
   menuButton.classList.add('hide');
   menuCloseButton.classList.remove('hide');
+  background.classList.add('gray-1');
```
Comment (cwefso) [inline src/main.js:62]: adds a new class called gray-1 to the new section

File: styles.css [hunk 4]
```diff
@@ -345,6 +345,16 @@ body {
     display: flex;
     flex-direction: column;
     justify-content: space-between;
+    z-index: 2;
+  }
+
+  .gray-1 {
```
Comment (cwefso) [inline styles.css:351]: I used z-index to make this element appear in front of the others by setting it to an index of 1 and purple-4 and menu-body-mobile to 2. The higher the number the closer to the "front" of the page the element will be.

**Review by st0nebreaker — COMMENTED** [review]
Thank you for the annotations! Looks good

**Pull request merged**